advanced/mathematical_optimization/index.rst

```diff
@@ -886,8 +886,7 @@ While it is possible to construct our optimization problem ourselves,
 scipy provides a helper function for this purpose:
 :func:`scipy.optimize.curve_fit`::
 
-.. Comment to make doctest pass
-    >>> np.random.seed(0)
+    >>> np.random.seed(0) #doctest: +SKIP
```

Comment:
The whole goal of this comment was to 1) be invisible, 2) be executed by the doctest runner to make the doctest pass (by setting the random state). I think that the right fix is not to skip it, but to keep it hidden in a restructured text comment and move it elsewhere so that it does not break the indentation (as it is currently doing).